### DISCUSSION TITLE
am I able to Importing from file (CSV or JSON) & include image/file fields URL?

### DISCUSSION THREAD

**Discussion by eliteproxy7** [directus/directus#6525] [Archive]
can I specify an Image URL path or some other way to upload the images along with the item upload via JSON / CSV?

**Comment by rijkvanzanten** [comment]
No, that's not currently possible 👍🏻

**Comment by bezbeli** [comment]
Can this be done in any way? It would be great to be able to import json with urls and attach them as thumbnails.
